src/components/zones/ZonesView.tsx

```diff
@@ -7,10 +7,9 @@ import {
   useDisclosure,
 } from "@chakra-ui/react";
 import { css, cx } from "@emotion/css";
-import { useDispatch } from "react-redux";
+import { useDispatch, useSelector } from "react-redux";
```

Comment:
```suggestion
import { useDispatch } from "react-redux";
```
useSelector does not seem to be used